### DISCUSSION TITLE
Added the ability to use Custom CSS in Widgets

### DISCUSSION THREAD

**Discussion by RedstoneLP2** [baffler/Transparent-Twitch-Chat-Overlay#131] [Show and tell]
So, I added the ability to use Custom CSS in Widgets. Haven't fully tested it yet, but it looks to be working. Feel free to tell me what could be improved. Will probably open a Pull request soon, just wanted to try out Discussions. https://github.com/RedstoneLP2/Transparent-Twitch-Chat-Overlay

**Comment by baffler** [comment]
I'll check it out soon and let you know 👍
